module/proto/peggy/v1/query.proto

```diff
@@ -81,6 +85,10 @@ message QueryLastValsetRequestsResponse { repeated Valset valsets = 1; }
 message QueryLastPendingValsetRequestByAddrRequest { string address = 1; }
 message QueryLastPendingValsetRequestByAddrResponse { repeated Valset valsets = 1; }
 
+message QueryTokenFeeMapRequest {}
+message QueryTokenFeeMapResponse { map<string, string> tokenFeeMap = 1; }
```

Comment:
Strings seem like a poor choice of data type for this. Why should the caller have to parse all the info they may want out of this endpoint?

Ideally we use the message types to enforce safety and reduce potential programmer error. I'm actually somewhat put out that we don't have a highly validated ethereum address type in the proto and would like to use sdk.AccAddress and sdk.ValAddress instead of strings where possible.

This should also probably be renamed 'batch fees' because we have arbitrary logic coming up and it's not subject to the same rules batches are when it comes to fees. Batches must have a fee type matching the token type that batch is transferring. Whereas arbitrary logic calls have an arbitrary number of arbitrary erc20's as fees.

BatchFees {
token: String
totalInPool: sdk.Int
topOneHundred: sdk.Int
}

BatchFeeResponse {
repeated batchFees
}

Notice that I'm separating the total in the pool versus the top 100, since a batch when requested will grab only the top 100 that's the number you want to know for 'is my next batch going to be profitable?'
The total in the pool gives you an idea of how much remains, maybe requesting multiple batches is a good idea.

You can omit the total in the pool if you like, it's non-essentail, but the endpoint must answer the question of 'should I request a batch of this token type?'